### PR TITLE
Implement API to test mail

### PR DESCRIPTION
**Pull request by Abyss777**
- added API point `api/users/notifications/testmail` that sends "unknown" event on GET request
- make `sendMailSync` public
- moved try/catch to runable

I decided to throw exceptions in resource. It will help debug without looking in log file.

### CODE REVIEW

File: src/org/traccar/api/resource/NotificationResource.java [hunk 1]
```diff
@@ -57,4 +60,11 @@ public Response update(Notification entity) throws SQLException {
         return Response.ok(entity).build();
     }
 
+    @Path("testmail")
```
Comment (tananaev) [inline src/org/traccar/api/resource/NotificationResource.java]: I think we should use "POST" method, and we can probably name it just "test".

File: src/org/traccar/api/resource/NotificationResource.java [hunk 2]
```diff
@@ -57,4 +60,11 @@ public Response update(Notification entity) throws SQLException {
...
+    @GET
+    public Response testMail() throws MessagingException {
+        NotificationMail.sendMailSync(getUserId(), new Event("unknown", 0), null);
```
Comment (tananaev) [inline src/org/traccar/api/resource/NotificationResource.java]: Maybe use "test" instead of "unknown"?

Comment (Abyss777) [inline src/org/traccar/api/resource/NotificationResource.java]: Frankly, it does not really mater. Just should disagree with existent types.
The email will come with "Unknown type" text.

**Comment by Abyss777** [comment]
Renamed and used POST

**Comment by tananaev** [comment]
Merged, thanks.

**Comment by Abyss777** [comment]
Have no idea where to put button in web interface. Do you have?

**Comment by tananaev** [comment]
I was thinking we can put it in the user dialog.

**Pull request merged**